honeybee_3dm/togeometry.py

```diff
@@ -166,7 +162,10 @@ def brep_to_meshed_face3d(brep, tolerance):
     for i in range(len(brep.Faces)):
         mesh = brep.Faces[i].GetMesh(rhino3dm.MeshType.Any)
         faces.extend(mesh_to_face3d(mesh))
-    polyface = Polyface3D.from_faces(faces, tolerance)
+    try:
+        polyface = Polyface3D.from_faces(faces, tolerance)
+    except Exception:
+        raise ValueError
```

Comment:
You should include a helpful error message here. This will be confusing.

raise ValueError('your message here!')

Also what is the type of exception that you're trying to address here? A generic Exception is almost always a bad idea.